Danielle/updates

dnkem opened the pull request:
# Updated and Completed the Mobile and Desktop NavBar CSS
- replace this with a short description of what the pr is for

# Changes
- adjusted the structure of the nav bar and got rid of the contact us button
- updated css

# Additional Info
- any additional information or context

# Screenshots (if necessary)
<img width="1247" alt="Screenshot 2024-12-19 at 12 34 50 AM" src="https://github.com/user-attachments/assets/0750961d-0473-4b56-a554-c52f177f7746" />
<img width="340" alt="Screenshot 2024-12-19 at 12 35 33 AM" src="https://github.com/user-attachments/assets/16c63cfd-e84b-4211-ae42-7f1733b154a7" />
Original | Updated
:------------------------:|:------------------------:
original screenshot | updated screenshot

Kanik-a committed:
Create individual_events_page.astro

ney927 committed:
Merge branch 'main' into Individual-Events-Page

ney927 committed:
updates

implemented page styling
use dynamic url instead of queries

dnkem committed:
Made some changes to desktop nav

dnkem committed:
Updates to the mobile and website navbar

dnkem committed:
Added in the rest of About us page

dnkem committed:
Removed the bottom stars because they are causing the issue with the black side bar in this instance

ney927 committed:
fix black bar on screen (problem was Dropdown)

ney927 committed:
Merge branch 'Individual-Events-Page' into danielle/updates

ney927 committed:
updates

fix individual event card page for new poster size
make footer 'made with <3' img smaller
add 'No ___ Events' to events main page

ney927 committed:
add resources link

ghost commented:
The preview deployment is ready. 🟢

[Open Preview](https://62.technolgap.dev.cublueprint.org) | [Open Build Logs](https://develop.cublueprint.org/project/lgggwkk4kc4ks80800c0408o/production/application/bgsoowcc0socsksgcoow0sk0/deployment/twww8gwc8cgcg80gooo8k880)


Last updated at: 2024-12-25 06:18:45 CET

ney927 committed:
change navbar links

ney927 committed:
navbar height decreases for mobile

ney927 committed:
Update [event_id].astro

ney927 committed:
navbar positioning small fixes

ney927 committed:
updates

fix stylings on events pages
add speaker to navbar
add links to speaker/collab/spons on indivudual events page

ney927 committed:
black bar on events  main page and series page

ney927 committed:
fix resources page responsiveness

ney927 committed:
fix homepage mobile
